xml/System.Text.Json.Serialization/JsonConverter.xml

```diff
@@ -37,11 +37,11 @@
         <Parameter Name="typeToConvert" Type="System.Type" />
       </Parameters>
       <Docs>
-        <param name="typeToConvert">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="typeToConvert">The type to compare against.</param>
+        <summary>When overridden in a derived class, determines whether the converter can convert the specified type.</summary>
+        <returns><see langword="true" /> if the type can be converted; otherwise, <see langword="false" />.</returns>
```

Comment:
I like how Json.NET words it: https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonConverter_CanConvert.htm

```suggestion
        <returns><see langword="true" /> if this instance can convert the specified object type; otherwise, <see langword="false" />.</returns>
```